tsconfig.paths.json

```diff
@@ -3,7 +3,19 @@
     "baseUrl": ".",
     "paths": {
       "@Pages": ["src/pages"],
-      "@Pages/*": ["src/pages/*"]
+      "@Pages/*": ["src/pages/*"],
+      "@Constants": ["src/constants"],
+      "@Constants/*": ["src/constants/*"],
+      "@SharedComponents": ["src/components"],
+      "@SharedComponents/*": ["src/components/*"],
+      "@Services": ["src/services"],
+      "@Services/*": ["src/services/*"],
+      "@HOCs": ["src/HOCs"],
+      "@HOCs/*": ["src/HOCs/*"],
+      "@Images": ["src/images"],
+      "@Images/*": ["src/images/*"],
+      "@Contexts": ["src/contexts"],
+      "@Contexts/*": ["src/contexts/*"]
```

Comment:
src/contexts is a file, not a folder, so idk if you can do src/contexts/*

Reply:
Besides this, this PR should be good. I'll approve when you're done fixing this little thing.

Reply:
just merge it in for me as well